docs/src/main/asciidoc/spring-cloud-config-client.adoc

```diff
@@ -0,0 +1,140 @@
+
+include::./status-include.adoc[]
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+
+== Stand up a Config Server
+
+To stand up a Config Server required for this guide, please follow the instructions outlined https://github.com/spring-guides/gs-centralized-configuration#stand-up-a-config-server[here].
+The end result of that process is a running Config Server that will provide the `Hello world` value for a configuration property named `message` when the application querying the server is named `a-bootiful-client`.
+
+== Creating the Maven project
+
+First, we need a new project. Create a new project with the following command:
+
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=spring-cloud-config-quickstart \
+    -DclassName="org.acme.spring.cloud.config.client.GreetingResource" \
+    -Dpath="/greeting" \
+    -Dextensions="spring-cloud-config-client"
+cd spring-cloud-config-quickstart
+----
+
+This command generates a Maven project with a REST endpoint and imports the `spring-cloud-config-client` extension.
+
+== GreetingController
+
+The Quarkus maven plugin automatically generated a `GreetingResource` JAX-RS resource in the
+`src/main/java/org/acme/spring/cloud/config/client/GreetingResource.java` file that looks like:
+
+[source,java]
+----
+package org.acme.spring.spring.cloud.config.client;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        return "hello";
+    }
+}
+----
+
+As we want to use configuration properties read from the Config Server, we will update the `GreetingResource` to inject the `message` property. The updated code will look like this:
+
+[source,java]
+----
+package org.acme.spring.spring.cloud.config.client;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @ConfigProperty(name = "message", defaultValue="hello default")
+    String message;
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        return message;
+    }
+}
+----
+
+== Configuring the application
+
+Quarkus provides various configuration knobs under the `quarkus.scccc` root. For the purposes of this guide, our Quarkus application is going to be configured in `application.properties` as follows:
```

Comment:
> `quarkus.scccc`

Hum. I totally missed in the initial commit but it looks very cryptic. Can't we find a better name for this? Or is it well known in the Spring world?

`quarkus.spring-cloud-config` would be better as a root, no?

Reply:
This is just something I picked so it wouldn't be super long to type. I'm definitely not tied to this, so I am open to ideas. @gastaldi what's your take?

Reply:
I went with your suggestion. But if anyone has any other suggestion, I am willing to hear it :)

Reply:
+1